core/field_label.js

```diff
@@ -40,22 +40,30 @@ goog.require('Blockly.utils.Size');
  * @param {string=} opt_value The initial value of the field. Should cast to a
  *    string. Defaults to an empty string if null or undefined.
  * @param {string=} opt_class Optional CSS class for the field's text.
+ * @param {Object=} opt_config A map of options used to configure the field.
+ *    See the [field creation documentation]{@link https://developers.google.com/blockly/guides/create-custom-blocks/fields/built-in-fields/label#creation}
+ *    for a list of properties this parameter supports.
  * @extends {Blockly.Field}
  * @constructor
  */
-Blockly.FieldLabel = function(opt_value, opt_class) {
+Blockly.FieldLabel = function(opt_value, opt_class, opt_config) {
+  if (opt_value == null) {
+    opt_value = '';
+  }  // Else it gets cast to a string (e.g. false -> 'false').
+
   /**
    * The html class name to use for this field.
-   * @type {?string}
+   * @type {string}
    * @private
    */
-  this.class_ = opt_class || null;
+  this.class_ = '';
```

Comment:
Why not keep this as a nullable string?
That way we don't write an empty class to the dom element.

Reply:
ie: set it to ``null`` here.

Reply:
Oh nevermind, your check won't write a class if it's empty.
Still I don't see any reason to make it an empty string.